### PR TITLE
Fix page load logic

### PR DESCRIPTION
**Pull request by imurchie**
Reproduce #197 with latest code.

### CODE REVIEW

File: lib/commands/context.js [hunk 1]
```diff
@@ -317,13 +317,13 @@ extensions.onPageChange = async function (pageChangeNotification) {
       let needsPageLoad = (() => {
         let item = (arr) => {
           return _.filter(arr, (obj) => {
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: wouldn't be https://lodash.com/docs/4.17.4#find more appropriate for such purpose?

Comment (imurchie) [reply lib/commands/context.js]: It might do the same thing, yes. Not sure about "more appropriate".

Comment (imurchie) [reply lib/commands/context.js]: Trying it out...

**Comment by imurchie** [comment]
Published in `appium-ios-driver@1.27.4`.

**Pull request merged**